src/app/config.rs

```diff
@@ -314,6 +320,7 @@ mod tests {
                 ("UPTIME_CHECKER_CHECKER_NUMBER", "2"),
                 ("UPTIME_CHECKER_TOTAL_CHECKERS", "5"),
                 ("UPTIME_CHECKER_FAILURE_RETRIES", "2"),
+                ("UPTIME_CHECKER_HTTP_CHECKER_DNS_NAMESERVERS", "8.8.8.8,8.8.4.4")
```

Comment:
do we know if google's public DNS servers do any kind of rate limiting?

Reply:
Not sure, This is just a test that is reading the values though